tests/integration/helpers/web-snippet-test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | web-snippet', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
+  test('it renders', async function(assert) {
+    this.set('inputValue', [
+      {
+        primary: {
+          code: [
+            {text: '1234'},
+          ],
```

Comment:
Peut-être serait-il plus correct de tester le `map` en donnant plusieurs `code` et en testant leur jointure:
```js
code: [
  { text: '1234' },
  { text: '5678' },
],
```
--> `assert.equal(this.element.textContent.trim(), '1234 5678');`

Reply:
Il n'y a qu'un seul text par code